### DISCUSSION TITLE
Open Source Diversity

### DISCUSSION THREAD

**Discussion by pdurbin** [bostonopen/bostonopen.github.io#10] [General]
I just noticed that @opensourcediversity lists local meetups at https://opensourcediversity.org/#spaces

<img width="1582" alt="Screenshot 2023-09-24 at 1 05 47 PM" src="https://github.com/bostonopen/bostonopen.github.io/assets/21006/ef8c9a12-d668-4f14-be70-d9996a443134">

Should we ask to be listed? Here's an example PR to add a meetup:

- https://github.com/opensourcediversity/opensourcediversity.org/pull/66

I think we could simply create an issue or a PR or start a discussion first at https://discourse.opensourcediversity.org

**Comment by HarshKapadia2** [comment]
Maybe we need more people attending to list ourselves there? No clue. Asking them would be better!
